app/templates/app/role/index.hbs

```diff
@@ -8,22 +8,33 @@
       <div class="col-md-12">
```

Comment:
Lets define this such that it will well on different screen sizes, as the moment on tablet or mobile view this is not very responsive.